app/liquid_tags/user_subscription_tag.rb

```diff
@@ -147,9 +147,11 @@ class UserSubscriptionTag < LiquidTagBase
     // ***************************************
     function addSignInClickHandler() {
       const signInBtn = document.getElementById('sign-in-btn');
-      if (signInBtn && typeof showModal !== 'undefined') {
+      if (signInBtn) {
         signInBtn.addEventListener('click', function(e) {
-          showModal('email_signup');
+          if (typeof showModal !== 'undefined') {
+            showModal('user_subscription');
+          }
```

Comment:
Huh, I guess this is because of some async stuff going on? I would think the original would work but I guess not...

Reply:
Right!? I thought the exact same thing 🙈 . The original kept failing because `showModal` was undefined.

I think changing it back to this way works because it's not checking for `showModal` until "click" and by that point, everything is loaded.

Reply:
Ahh that makes a lot of sense!